Events-Team.md

```diff
@@ -10,4 +10,25 @@ If you are interested in helping out with events in Oppia and want to join the t
 
 ## Gaming Event server
 
-Oppia has a gaming event server on discord where we usually join for fun nights playing Among Us, Chess, Skribl, etc. Here is a link to join the server: [https://discord.com/invite/dCJwEAgk3D](https://discord.com/invite/dCJwEAgk3D).
\ No newline at end of file
+Oppia has a gaming event server on discord where we usually join for fun nights playing Among Us, Chess, Skribl, etc. Here is a link to join the server: [https://discord.com/invite/dCJwEAgk3D](https://discord.com/invite/dCJwEAgk3D).
+
+### Recording events
```

Comment:
Do you think it would be a good idea to put this under `Events` since it currently looks like it falls under `Gaming Event server`

Reply:
Done.